Dockerfile

```diff
@@ -19,7 +19,8 @@ FROM base as production
 RUN pip install gunicorn
 COPY todo_app todo_app 
 EXPOSE 5000
-ENTRYPOINT [ "gunicorn","--bind","0.0.0.0:5000","todo_app.app:wsgi_c"]
+# ENTRYPOINT [ "gunicorn","--bind","0.0.0.0:${PORT}", "todo_app.app:wsgi_c"]
+CMD gunicorn --bind 0.0.0.0:$PORT todo_app.app:wsgi_c
```

Comment:
Your instructions in the README for how to run the application no longer work, because $PORT is **required**

You can set a default value either with a Dockerfile command `ENV PORT=5000` or you could use `${PORT:-5000}`

Reply:
I am now passing value for PORT through docker run command if running on local.